tests/benchmarks/multicore/idle_uarte/prj.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_SERIAL=y
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_UART_USE_RUNTIME_CONFIGURE=y
+CONFIG_UART_ASYNC_API=y
+
+CONFIG_PM=y
+CONFIG_PM_S2RAM=y
+CONFIG_PM_S2RAM_CUSTOM_MARKING=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_POWEROFF=y
+
+CONFIG_GPIO=n
+CONFIG_BOOT_BANNER=n
+CONFIG_NRFS_MRAM_SERVICE_ENABLED=n
+CONFIG_CLOCK_CONTROL=n
+
+# Enable for debugging purposes only
+CONFIG_PRINTK=n
+CONFIG_LOG=n
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
```

Comment:
not needed

Reply:
This is left here for convenience only, if debugging is desired.